resources/public/api/conf/P1.0/application.raml

```diff
@@ -34,27 +34,118 @@ uses:
               application/json:
                 example: "\"Hello World\""
 
+    /:
+      get:
+        displayName: Root
+        description: Use this endpoint to get a list of all available endpoints for Individuals Details.
+        is: [headers.acceptHeader]
```

Comment:
I think we will need to keep the annotations. 

See here:

https://github.com/hmrc/individuals-employments-api/pull/55/files#diff-c09c12b618659dd058ee4ffdac7c65b47450ae228d9c7af5022a0ef86ce225c0R33